css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+#card-deck {
+  display: flex;
+  flex-wrap: wrap;
+  flex-basis: calc(100% / 6);
```

Comment:
flex basis is a flex item property, so it only needs to be placed on the item you want to set width on.